### PR TITLE
gh-117 toResult cover with unit-tests

### PR DESCRIPTION
**Pull request by oleksii-yurchenko**
- result.status returns 1
- res.send called with result
- toResult returns result

### CODE REVIEW

File: packages/util-use-case/lib/adapters/toResult.test.mjs [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+import {toResult} from './toResult'
+
+const result = {}
+const req = {}
+const res = {send: sinon.stub()}
+const deps = {}
+
+const toResultReturns = toResult(result, req, res, deps)
+
+test('Test toResult func', (t) => {
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toResult.test.mjs]: `Test toResult func` it's too verbose, it makes sense to highlight that it's a simple scenario, for example`basic happy path`

File: packages/util-use-case/lib/adapters/toResult.test.mjs [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import sinon from 'sinon'
+
+import {toResult} from './toResult'
+
+const result = {}
+const req = {}
+const res = {send: sinon.stub()}
+const deps = {}
+
+const toResultReturns = toResult(result, req, res, deps)
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toResult.test.mjs]: strange naming, why not `result = toResult(result, req, res, deps)`, also move that expression to `test(...)`

or 

```
const expected = ...
actual  = toResult(result, req, res, deps)

// compare  expected and actual here
```

File: packages/util-use-case/lib/adapters/toResult.test.mjs [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+test('Test toResult func', (t) => {
+  t.assert(result.status === 1, 'Added new prop status 1')
+  t.assert(res.send.calledOnceWith(result), 'res.send has been called with result')
+  t.assert(toResultReturns === result, 'toResult returns result')
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toResult.test.mjs]: 'returns expected result'

File: packages/util-use-case/lib/adapters/toResult.test.mjs [hunk 4]
```diff
@@ -0,0 +1,17 @@
+import test from '@zorko-io/tool-test-harness'
+import sinon from 'sinon'
+
+import {toResult} from './toResult'
+
+const result = {}
```
Comment (nesterone) [inline packages/util-use-case/lib/adapters/toResult.test.mjs]: move definitions like 

```
const result = {}
const req = {}
const res = {send: sinon.stub()}
const deps = {}



```

under test(...)

or use `test.beforeEach` for initialization of test inputs

**Review by nesterone — CHANGES_REQUESTED** [review]

File: packages/util-use-case/lib/adapters/toResult.test.mjs (hunk 2 above)
Comment (oleksii-yurchenko) [reply packages/util-use-case/lib/adapters/toResult.test.mjs]: because we already have variable result.

**Review by nesterone — APPROVED** [review]

**Pull request merged**